### PR TITLE
feat(cli): add CRUD application blueprint

### PR DESCRIPTION
**Pull request by killiankopp**
## Pourquoi

Le CRUD est un cas applicatif fréquent, mais ce n'est ni le comportement universel d'une entité, ni une capability, ni un adapter. Cette PR introduit donc la notion générique de **blueprint applicatif** et livre CRUD comme première implémentation, sans coupler le cœur métier à FastAPI, FastMCP ou à une base de données.

Cette livraison constitue la première implémentation concrète du chantier #155.

## Ce que livre cette PR

- ajoute le catalogue `arclith-cli blueprints` et sa sortie `--json` ;
- ajoute `arclith-cli add-blueprint crud --entity <Entity>` avec `--feature` et `--dry-run` ;
- propose `minimal` ou `crud` lors de la création interactive d'une entité ;
- ajoute `--profile minimal|crud` à `add-entity` et `new` pour les scripts, agents et CI ;
- génère les ports inbound et use cases `create`, `get`, `list`, `update`, `delete`, les erreurs applicatives, le container de composition, un test complet et une documentation locale ;
- écrit le manifeste strict et versionné `.arclith/features/<feature>.yaml` ;
- enregistre la version, les opérations et l'empreinte réelle des templates dans `arclith.recipe.yaml` ;
- rend `add-blueprint` et les profils rejouables, tout en conservant la compatibilité des anciennes recettes `add-entity` sans profil ;
- refuse les collisions lors d'une première installation, puis préserve le code développeur lors des replays ;
- documente explicitement la séparation blueprint applicatif / capability / adapter.

## Architecture retenue

```text
Entity
  -> Application blueprint (CRUD aujourd'hui)
     -> ports + use cases + composition + tests
        -> adapter(s) choisi(s) séparément
           -> FastAPI, FastMCP, RabbitMQ, MongoDB, PostgreSQL, ...
```

Le blueprint compose les primitives CRUD existantes d'Arclith via `BaseService[T]`. Il ne génère aucune route, aucun tool MCP et aucun adapter de persistence. Les erreurs `NotFound` et `VersionConflict` restent applicatives afin que chaque transport choisisse sa représentation publique.

`duplicate` et `purge` ne sont pas inclus : ce sont des comportements explicites, pas des opérations CRUD.

## Validation

- `uv run --project cli ruff format --check cli/arclith_cli cli/tests`
- `uv run --project cli ruff check cli/arclith_cli cli/tests`
- `.venv/bin/python -m mypy cli/arclith_cli`
- `uv run --project cli pytest -q cli/tests` : **294 passed**
- `make precommit` : Ruff, mypy et Bandit réussis
- `make coverage` : **2235 passed, 5 skipped, 91.19%**
- `make docs` : build MkDocs strict réussi
- `uv build cli` : sdist et wheel réussis
- projet frais généré : Ruff réussi, compilation réussie et **3 tests projet réussis**, dont le cycle CRUD complet

## Hors périmètre de cette itération

La projection automatique du manifeste vers FastAPI, FastMCP, RabbitMQ ou un autre transport n'est pas incluse. Elle formera l'itération suivante : elle devra consommer `.arclith/features/<feature>.yaml` après installation explicite du transport et générer uniquement les bindings demandés.

Le contrôle de version généré détecte une version obsolète dans le flux applicatif, mais le port `Repository[T]` générique ne promet pas un compare-and-swap atomique multi-processus. Un besoin métier de cette nature doit rester un port spécialisé avec une implémentation transactionnelle adaptée au store.

### CODE REVIEW

File: cli/arclith_cli/blueprint_generation.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    rendered = render_application_blueprint(blueprint, paths, entity, feature)
+    invalid_targets = sorted(
+        path for path in rendered if path.exists() and not path.is_file()
+    )
+    if invalid_targets:
+        relative = ", ".join(
+            str(path.relative_to(project_dir)) for path in invalid_targets
+        )
+        raise ValueError(f"Application blueprint target is not a file: {relative}")
+    installed = manifest_path.is_file()
```
[the remark this answers is not in the document]
Comment (killiankopp) [reply cli/arclith_cli/blueprint_generation.py:104]: Fixed in 9c37033: the manifest path now participates in the non-file preflight before any generated source is written. Added a regression proving that an existing manifest directory rejects the plan without creating CRUD files.

File: cli/arclith_cli/feature_manifest.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    @classmethod
+    def from_dict(cls, raw: object) -> FeatureManifest:
+        data = _mapping(raw, "feature manifest")
+        _exact_keys(
+            data,
+            {"version", "feature", "entity", "blueprint", "operations"},
+            "feature manifest",
+        )
+        version = data["version"]
+        if version != FEATURE_MANIFEST_VERSION:
```
[the remark this answers is not in the document]
Comment (killiankopp) [reply cli/arclith_cli/feature_manifest.py:74]: Fixed in 9c37033: the top-level feature manifest version now rejects bool and every non-int before comparing the supported schema version. A version:true regression test covers the Python bool/int edge case.

File: cli/arclith_cli/feature_manifest.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+            "entity": self.entity.to_dict(),
+            "blueprint": self.blueprint.to_dict(),
+            "operations": list(self.operations),
+        }
+
+
+def load_feature_manifest(path: Path) -> FeatureManifest:
+    if not path.is_file():
+        raise ValueError(f"Feature manifest not found: {path}")
+    raw = yaml.safe_load(path.read_text(encoding="utf-8"))
```
[the remark this answers is not in the document]
Comment (killiankopp) [reply cli/arclith_cli/feature_manifest.py]: Fixed in 9c37033: yaml.YAMLError is translated into a contextual ValueError, which the add-blueprint CLI boundary already renders as a clean refusal. Added malformed-YAML coverage.

**Comment by killiankopp** [comment]
Revue Copilot traitée dans `9c37033` :

- préflight du profil avant création de l’entité lors du rejeu, y compris pour les collisions ;
- rejet d’un chemin de manifeste qui est un répertoire avant toute écriture ;
- rejet explicite des versions YAML booléennes ;
- conversion des erreurs YAML en erreurs de validation CLI propres ;
- rejet cohérent des noms normalisés en mots-clés Python réservés.

Régressions ajoutées pour chaque cas. Validations locales : 301 tests CLI, 2 235 tests framework + 5 ignorés, couverture 91,19 %, Ruff, mypy, Bandit, MkDocs strict et build du package réussis.
